### PR TITLE
Infrastructure: Add HTMLHint

### PR DESCRIPTION
**Pull request by nschonni**
- Add HTMLHint that catched a few more things than the validator
- Add explicit closing tags for lists and table items
- Use double quotes for `data` values
- Ignore the camel casing in `viewBox` SVG property

***
<a href="https://pr-preview.s3.amazonaws.com/nschonni/aria-practices/pull/1608.html" title="Last updated on Dec 14, 2020, 4:08 AM UTC (d44e17b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria-practices/1608/f8cdf1c...nschonni:d44e17b.html" title="Last updated on Dec 14, 2020, 4:08 AM UTC (d44e17b)">Diff</a>

### CODE REVIEW

**Comment by nschonni** [comment]
Seems like the "preview" is OK, but the "diff" view breaks the HTML in the code blocks. Not sure if that's a known issue as I got a warning locally to update the respec version, but I didn't want to roll that in here

**Comment by spectranaut** [comment]
Do we need both vnu-jar and this linter? Do you know if this new linter covers all the vnu-jar cases?

**Comment by nschonni** [comment]
I think the vnu-jar is still good to keep. The HTMLHint seems to have a smaller set of rules that it checks https://htmlhint.com/docs/user-guide/list-rules, but since it is a Hinter/Linter, it flags a few things that are "valid" but maybe mistakes/inconsistencies

**Review by spectranaut — APPROVED** [review]
Then I approve this PR :)

File: package.json [hunk 1]
```diff
@@ -8,11 +8,12 @@
   },
   "scripts": {
     "fix": "npm run lint:es -- --fix && npm run lint:css -- --fix",
+    "htmlhint": "htmlhint \"**/*.html\" -i \"common/**/*.html\" -f unix",
     "lint": "npm run lint:es && npm run lint:css && npm run vnu-jar && npm run lint:spelling",
```
Comment (spectranaut) [inline package.json:12]: this should call `npm run lint:html` instead of `npm run vnu-jar`!

**Review by spectranaut — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (nschonni) [reply package.json]: Good call! fixed

**Review by spectranaut — APPROVED** [review]
thanks for that fix :) looks good to land!

**Comment by mcking65** [comment]
@nschonni 
I merged that htmlparser2 PR and I think that caused a conflict here. Mind fixing when you have chance?

**Comment by nschonni** [comment]
@mcking65 rebased

**Comment by mcking65** [comment]
Hmm, @nschonni 
It looks like the new Check examples/index.html is generating a sperious failure. If I'm understanding the diff, it might be failing on an end of file marker difference.

**Review by mcking65 — APPROVED** [review]
Looks good to go, thank you for the rebase!

**Pull request merged**